pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -305,7 +305,7 @@ func (pl *dynamicResources) podResourceClaims(pod *v1.Pod) ([]*resourcev1alpha2.
 // the pod cannot be scheduled at the moment on any node.
 func (pl *dynamicResources) PreFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod) (*framework.PreFilterResult, *framework.Status) {
 	if !pl.enabled {
-		return nil, nil
+		return nil, framework.NewStatus(framework.Skip)
```

Comment:
Thanks @pohly 
I don't think we can assume that the status object won't be modified by the caller, for example:
https://github.com/kubernetes/kubernetes/blob/b53411ffb813b863da12f200b0cfa9c6b3847cb8/pkg/scheduler/framework/runtime/framework.go#L649-L655

So it is safer to use independent variables in this case. Shared variable can make troubleshooting difficult.

Reply:
Also, we already used a similar approach on other plugins. This may be a good optimization point, but it seems like something more should be considered.

Reply:
Then this PR looks good to me.